### PR TITLE
Fix for JENKINS-26907

### PR DESCRIPTION
**Pull request by darraghs**

### CODE REVIEW

**Comment by darraghs** [comment]
@Brantone Can you please review this PR and commit it? Would like to get this fixed

**Comment by Brantone** [comment]
Yup, reviewing now. Sorry for delay, transitioned gigs recently and lost access to instance of HockeyApp to test :(

File: src/main/java/hockeyapp/HockeyappRecorder.java [hunk 1]
```diff
@@ -82,7 +78,6 @@
     @DataBoundConstructor
     public HockeyappRecorder(List<HockeyappApplication> applications, boolean debugMode,
                              BaseUrlHolder baseUrlHolder, boolean failGracefully) {
-        this.schemaVersion = SCHEMA_VERSION_NUMBER;
```
Comment (Brantone) [inline src/main/java/hockeyapp/HockeyappRecorder.java]: Seems odd that removing the version would fix, but more likely to believe team of people at Jenkins World :)

Comment (darraghs) [inline src/main/java/hockeyapp/HockeyappRecorder.java]: From my memory, I think the schema version was serialised to the file but could not be read back in and this cause part of the issue

**Pull request merged**